backend/config/curation-api.yml

```diff
@@ -154,6 +154,8 @@ paths:
       description: >
         Fetch full metadata for the Collection and its Datasets.
 
+        If an Authentication token is provided and the curator is allowed, then revision-related fields will also be returned.
```

Comment:
> revision-related fields

This is slightly confusing. Was this specifically requested by Brian or Jason?

Reply:
yes, they wanted to know what happens if an auth token is provided.

Reply:
Doesn't this comprise solely either the `revision_of` or `revising_in` fields? The current phrasing sort of suggests that it returns fields for the revision collection.

Reply:
```suggestion
        An Authentication token must be provided in order to retrieve the id of an associated Revision for this Collection, if such a Revision currently exists.
```

Reply:
or something like that